CMakeModules/build_boost_compute.cmake

```diff
@@ -1,73 +1,22 @@
-# If using a commit, remove the v prefix to VER in URL.
```

Comment:
The ExternalProject_Add function was made to do this sort of thing. I don't know why we were doing this manually.